.github/workflows/ci.yml

```diff
@@ -6,10 +6,10 @@ on:
 
 jobs:
   full-build:
-    runs-on: macOS-latest
+    runs-on: macOS-12
```

Comment:
Lower in the file, you are using macos-11, please check what is appropriate version.

Reply:
Upgrading to 12 should be fine